src/services/useTimeSeries/index.ts

```diff
@@ -21,6 +21,7 @@ export interface UseTimeSeriesReturn {
   series: Ref<Record<string, Series>>
   isReady: Ref<boolean>
   isLoading: Ref<boolean>
+  loadingRequests: Ref<string[]>
```

Comment:
Maybe `loadingSeriesIds` or something? We have quite a lot of different IDs floating around so it's nice to know which one this array contains.